content/product/design/wildcard_design_system/index.md

```diff
@@ -65,3 +65,19 @@ When a change to an existing component is required, the designer who is proposin
   - Modify the main instance of the component in Wildcard
   - Publish changes with a fitting commit message
   - Add to the "Changelog" layer in the Wildcard Figma file
+
+## User interface patterns
+
+This section will detail user interface patterns not defined in the Wildcard Design System.
+
+### Getting started pages
+
+Getting started pages are presented to viewers of major Sourcegraph feature pages when they have not seen the page or when there is no content for the section. For un-authenticated users, the page serves as an overview and prompts the user login or sign up. For authenticated users, the page serves the same purpose and also provides helpful links to start using the feature.
+
+The following rules should be applied when selecting the page to display to users:
+
+- If the user has not seen the feature, the feature should display the getting started page
```

Comment:
```suggestion
- If the user has not seen the feature, the feature should display the getting started page
- If the user has not seen the feature but there is content available in this section, a modified getting started page should be displayed
```

This will be the case for code insights that users would see the feature for the first time but there will already be content to display. We're still deciding on how to present it.

Reply:
Awesome. what would the modifications to the getting started page be?

Also, I'm curious what content will exist in new installations?